docs/guides/server/configuration-production.adoc

```diff
@@ -83,4 +83,11 @@ For example, to change the IP stack preference to IPv4, set an environment varia
 export JAVA_OPTS_APPEND="-Djava.net.preferIPv4Stack=true"
 ----
 
+To set up the server for IPv6 only, set an environment variable as follows:
+
+[source, bash]
+----
+export JAVA_OPTS_APPEND="-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true"
+----
+
```

Comment:
I'm not sure if you wanna extend this information to a link to the JGroups docs: 
http://jgroups.org/manual5/index.html#IPAddresses

Section 5.3.3 may be relevant here.